cdc/owner/changefeed.go

```diff
@@ -513,6 +513,7 @@ LOOP:
 func (c *changefeed) releaseResources(ctx cdcContext.Context) {
 	if !c.initialized {
 		c.cleanupRedoManager(ctx)
+		c.cleanupServiceGCSafePoints(ctx)
```

Comment:
Just a side note: If the `cleanupServiceGCSafePoints` means cleaning up **temporary** service GC safe points, the function name should so indicate. Otherwise it would be confusing.